ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -153,6 +150,7 @@ StatusDialog {
         readonly property bool isSelectedHoldingValidAsset: !!selectedHolding && selectedHoldingType === Constants.TokenType.ERC20
 
         onSelectedHoldingChanged: {
+            d.sendError = ""
```

Comment:
Shouldn't you also clear the `sendError` on recipient change? I can't see that anywhere below

Reply:
Or maybe `onReCalculateSuggestedRoute` is simply enough for everything?